testrpc/cli.py

```diff
@@ -38,14 +45,15 @@ def runserver(host, port):
     for account in accounts:
         print('0x' + codecs.decode(codecs.encode(account, 'hex'), 'utf8'))
 
+    print("\nTransaction tracing is %s." % ( "enabled" if trace else "disabled"  ))
+
     print("\nListening on %s:%s" % (host, port))
 
     server = make_server(
         host,
         port,
         application,
     )
-
```

Comment:
This whitespace was here to visually separate these two lines.  Please re-add it.